playbooks/roles/atomic_registry/defaults/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+atomic_registry_upstream_tag: latest
+atomic_registry_upstream_image: "docker.io/projectatomic/atomic-registry-install:{{ atomic_registry_upstream_tag }}"
+atomic_registry_upstream_image_name: "projectatomic/atomic-registry-install:{{ atomic_registry_upstream_tag }}"
+
+atomic_registry_registry_image: docker.io/openshift/origin-docker-registry
+atomic_registry_registry_port: 5060
```

Comment:
Since this could be run on any box, not just a Katello box, I'd lean towards the standard defaults here as well.